index.html

```diff
@@ -64,5 +62,199 @@ <h1>
+                </div>
+            </section>
+
+
+            <section id="about-section" class="flex">
+                <div class="about-section-content">
+                    <h1>
+                        About<br>
+                        Myself
+
+                    </h1>
+                    <p>Hello I’m a software developer!
+                        I can help you build a product ,
+                        feature or website Look through some of my work and
+                        experience!
+                        If you like what you see and have a project you need
+                        coded,
+                        don’t hestiate to contact me.
+                    </p>
+                    <label class="lets-connect-label">LETS CONNECT</label>
+                    <ul>
+                        <li><span><a><img src="images/twitter-icon.svg"
+                                        alt="Twitter
+                                        Icon"></a></span></li>
+                        <li><span><a><img
+                                        src="images/linked-in-icon.svg"
+                                        alt="LinkedIn
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/medium-icon.svg"
+                                        alt="Medium
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/github-icon.svg"
+                                        alt="Github
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/angellist-icon.svg"
+                                        alt="Angellist
+                                        Icon"></a></span></li>
+                    </ul>
+                    <button class="styled-button" type="button">Get My Resume</button>
+                </div>
+                <ul id="about-section-dropdowns">
+                    <li class="flex language-icons-header-li">
+                        <label>Languages</label>
+                        <img src="images/dropdown-icon.png" alt="dropdown icon">
+                    </li>
+                    <li class="about-drop-down-icons-middle">
+                        <div>
+                            <img src="images/javascript-icon.png"
+                                alt="javascript icon">
+                            <label>javaScript</label>
+                        </div>
+                        <div>
+                            <img src="images/html-icon.png" alt="html icon">
+                            <label>HTML</label>
+                        </div>
+                        <div>
+                            <img src="images/css-icon.png" alt="css icon">
+                            <label>css</label>
+                        </div>
+                    </li>
+                    <li class="flex about-2-3-dropdowns">
```

Comment:
- [ ] the language section is not aligned correctly, as you can see this is the original 👇
![bb](https://user-images.githubusercontent.com/61562626/129423280-f235ac54-7d94-4e00-a688-a30fe3d43bbb.jpeg)

- And yours is like this 👇, the alignment is a bit off.
![nn](https://user-images.githubusercontent.com/61562626/129423297-940291fb-9520-4e6b-b50e-100bef5a3a94.jpeg)